server/sandbox_remove.go

```diff
@@ -41,7 +41,7 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 		if !sb.Stopped() {
 			cState := s.Runtime().ContainerStatus(c)
 			if cState.Status == oci.ContainerStateCreated || cState.Status == oci.ContainerStateRunning {
-				if err := s.Runtime().StopContainer(c, -1); err != nil {
+				if err := s.Runtime().StopContainer(ctx, c, 10); err != nil {
```

Comment:
Should the 10 be in a conf file somewhere on the very off chance we ever want to tweak it live?

Reply:
nope, nobody should tweak this and this PR is the same as #1010 so no need to re-review I guss